dds/DCPS/RTPS/ParameterListConverter.cpp

```diff
@@ -282,6 +282,20 @@ namespace {
 
     return OpenDDS::Security::DPDK_ORIGINAL;
   }
+
+  bool has_identity_status_token(const ParameterList& param_list)
+  {
+    CORBA::ULong length = param_list.length();
+    for (CORBA::ULong i = 0; i < length; ++i) {
+      const Parameter& param = param_list[i];
+      switch (param._d()) {
```

Comment:
Looks like it should be an `if` (also don't need a separately named `param` object)